.github/workflows/serving_publish.yml

```diff
@@ -61,14 +61,17 @@ jobs:
       - name: Publish snapshot deb
         if: ${{ github.event.inputs.mode == '' || github.event.inputs.mode == 'snapshot' }}
         run: |
-          ./gradlew :serving:createDeb -Psnapshot
+          ./gradlew :serving:distZip -Psnapshot
           DJL_VERSION=$(cat gradle.properties | awk -F '=' '/djl_version/ {print $2}')
           aws s3 cp serving/build/distributions/*.deb s3://djl-ai/publish/djl-serving/
-          aws cloudfront create-invalidation --distribution-id E371VB8JQ6NRVY --paths "/djl-serving/djl-serving_${DJL_VERSION}*"
-      - name: Publish release deb
+          aws cloudfront create-invalidation --distribution-id E371VB8JQ6NRVY --paths "/djl-serving/djl-serving_${DJL_VERSION}~SNAPSHOT*"
```

Comment:
We don't need add ~SNAPSHOT here, none SNAPSHOT version doesn't exist yet

Reply:
I can see non SNAPSHOT version like: [djl-serving_0.19.0-1_all.deb](https://s3.console.aws.amazon.com/s3/object/djl-ai?region=us-east-1&prefix=publish/djl-serving/djl-serving_0.19.0-1_all.deb)

Reply:
once we published 0.19.0, the snapshot build will be 0.20.0

Reply:
ok